File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/response/gropus/ItemsGroupsResponse.java
package com.xsolla.android.store.entity.response.gropus;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ItemsGroupsResponse {

    private List<Group> groups;

    private class Group {
        @SerializedName("external_id")
        private String externalId;
        private String name;
        private String description;
        @SerializedName("image_url")
        private String imageUrl;
        private int order;
        private int level;
        private List<Group> children;
        @SerializedName("parent_external_id")
        private String parentExternalId;
    }

}

File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/response/gropus/ItemsGroupsResponse.kt
package com.xsolla.android.store.entity.response.gropus

import com.google.gson.annotations.SerializedName

data class ItemsGroupsResponse(val groups: List<Group> = emptyList()) {
    data class Group(
        @SerializedName("external_id")
        val externalId: String? = null,
        val name: String? = null,
        val description: String? = null,
        @SerializedName("image_url")
        val imageUrl: String? = null,
        val order: Int,
        val level: Int,
        val children: List<Group> = emptyList(),
        @SerializedName("parent_external_id")
        val parentExternalId: String? = null
    )
}